modules/custom/az_finder/src/Plugin/views/exposed_form/QuickstartExposedFilters.php

```diff
@@ -0,0 +1,226 @@
+        'mb-3',
+      ]);
+      // Add the reset button visibility setting to the drupalSettings array.
+      if ($this->options['bef']['general']['reset_button_always_show'] === TRUE) {
+        $form['#attached']['drupalSettings']['azFinder']['alwaysDisplayResetButton'] = TRUE;
+      }
+      else {
+        $reset_button['#attributes']['class'][] = 'd-hidden';
+        $form['#attached']['drupalSettings']['azFinder']['alwaysDisplayResetButton'] = FALSE;
+      }
+      // Add the reset button counter setting to the drupalSettings array.
+      if ($this->options['reset_button_counter'] === TRUE) {
+        $form['#attached']['library'][] = 'az_finder/active-filter-count';
+        $count = [
+          '#type' => 'html_tag',
+          '#tag' => 'span',
+          '#attributes' => [
+            'class' => [
+              'js-active-filter-count',
+              'ml-1',
+            ],
+          ],
+        ];
+        $reset_button['count'] = $count;
+        $reset_button['#type'] = 'html_tag';
+        $reset_button['#tag'] = 'button';
+        $reset_button['#attributes']['type'] = 'button';
+        $reset_button['#attached']['library'][] = 'az_finder/active-filter-reset';
+        unset($reset_button['#pre_render']);
+      }
+      // Add the cloned reset button at the beginning of the form.
+      $form['top_reset'] = $reset_button;
+      // Hide the original reset button.
+      $form['actions']['reset']['#access'] = FALSE;
+    }
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function defineOptions() {
+    $options = parent::defineOptions();
+    $options['reset_button_position'] = ['default' => 'bottom'];
+    $options['reset_button_counter'] = ['default' => FALSE];
+    $options['orientation'] = ['default' => 'horizontal'];
+    $options['skip_link'] = ['default' => FALSE];
+    $options['skip_link_text'] = ['default' => $this->t('Skip to search and filter')];
+    $options['skip_link_id'] = ['default' => 'search-filter'];
+
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildOptionsForm(&$form, FormStateInterface $form_state) {
+    parent::buildOptionsForm($form, $form_state);
+    $reset_button_option = $form['bef']['general']['reset_button'];
+    unset($form['bef']['general']['reset_button']);
+    $form['bef']['general']['reset_button'] = $reset_button_option;
+    $form['bef']['general']['reset_button_settings'] = [
+      '#type' => 'details',
+      '#title' => $this->t('Reset Button Settings'),
+      '#open' => TRUE,
+      '#states' => [
+        'visible' => [
+          ':input[name="exposed_form_options[reset_button]"]' => ['checked' => TRUE],
+        ],
+      ],
+    ];
+    if (isset($form['bef']['general']['reset_button_always_show'])) {
+      $form['bef']['general']['reset_button_settings']['reset_button_always_show'] = $form['bef']['general']['reset_button_always_show'];
+      unset($form['bef']['general']['reset_button_always_show']);
+    }
```

Comment:
This does not show the button if unchecked and options on the front-end are selected.

Reply:
Also unable to check the "always show" option. Saved checkbox selection doesn't save.

Reply:
Fixed, I think. 🙇‍♂️